docs/maintainers/release.md

```diff
@@ -41,6 +41,13 @@ release. We use `<TAG>` as a placeholder for the release tag (e.g. `v1.4.0`).
      branch as `ANTREA_GIT_REVISION`. Test starting times need to be staggered:
      if multiple jobs run at the same time, the Jenkins worker may run
      out-of-memory.
+   - Kubernetes compatibility tests need to be triggered manually through
+     [the workflow](https://github.com/antrea-io/antrea/actions/workflows/conformance.yml)
+     to ensure the release is compatible with all [active K8s releases](https://kubernetes.io/releases/)
+     at the time of release. Click on `Run workflow`, enter the Antrea release
+     to test (e.g. `release-1.12`), enter the K8s versions to test (e.g.
+     `v1.27.2, v1.26.4, v1.25.9, v1.24.13`). You typically do not need to
```

Comment:
Do you mean we will also need to update the compatibility table with K8s patch release result?